src/misc/ethernet_pcap.cpp

```diff
@@ -126,7 +128,7 @@ bool PcapEthernetConnection::Initialize(void)
 	char errbuf[PCAP_ERRBUF_SIZE];
 	unsigned int userdev;
 #ifdef WIN32
-	if (pcap_findalldevs_ex(PCAP_SRC_IF_STRING, NULL, &alldevs, errbuf) == -1)
+	if (pcap_findalldevs_ex(pcap_src_if_string, NULL, &alldevs, errbuf) == -1)
```

Comment:
Suggest mass-cleanup of `NULL` and `0` pointers to `nullptr` in the DOSBox-side code that is being moved about in these commits (but suggest leaving the NE2000 driver as-is).